src/components/media-loader.js

```diff
@@ -402,7 +402,7 @@ AFRAME.registerComponent("media-loader", {
       }
 
       // Some servers treat m3u8 playlists as "audio/x-mpegurl", we always want to treat them as HLS videos
-      if (contentType === "audio/x-mpegurl") {
+      if (contentType === "audio/x-mpegurl" || contentType === "audio/mpegurl") {
```

Comment:
Not sure, but we might want to check `"application/x-mpegurl"`, `"application/mpegurl"`, and `"application/vnd.apple.mpegurl.audio"` as per https://en.wikipedia.org/wiki/M3U#Internet_media_types